c_src/absTab.h

```diff
@@ -0,0 +1,20 @@
+#ifndef ABSTAB_H
+#define ABSTAB_H
+
+#include <stdio.h>
+
+/* forward declarations */
+typedef struct AbsTab *AbsTab_t;
```

Comment:
To stay in line with the conventions in this code base I'd avoid typedef-ing away the `struct` keyword. 

This is also fairly subjective and up to personal preference, but I tend to not hide things like pointers in typedefs as it makes the interface less obvious.